From: Aron Xu <happyaron@ubuntu.com>
Subject: Do not check for wx to enable pyprompter
 Do not check for wx during configure time to enable
 pyprompter without adding python-wxgtk3.0 to build
 dependency, which is not wanted in main.
Index: presage-0.9/configure.ac
===================================================================
--- presage-0.9.orig/configure.ac
+++ presage-0.9/configure.ac
@@ -341,38 +341,7 @@ AM_CONDITIONAL([ENABLE_PYPRESAGEMATE], [
 dnl =====================
 dnl Checks for pyprompter
 dnl =====================
-if test "$PYTHON" != :
-then
-    AC_MSG_CHECKING(for python wx module)
-    $PYTHON -c "import wx" 2&>/dev/null
-    if test $? -eq 0;
-    then
-    	have_python_wx=yes
-    else
-    	have_python_wx=no
-    fi
-    AC_MSG_RESULT($have_python_wx)
-    
-    
-    AC_MSG_CHECKING(for python wx.stc module)
-    $PYTHON -c "import wx.stc" 2&>/dev/null
-    if test $? -eq 0;
-    then
-    	have_python_wx_stc=yes
-    else
-    	have_python_wx_stc=no
-    fi
-    AC_MSG_RESULT($have_python_wx_stc)
-fi
-
-if test "$PYTHON" != : -a \
-	"x$have_python_wx" = "xyes" -a \
-	"x$have_python_wx_stc" = "xyes"
-then
-	enable_pyprompter=yes
-else
-	enable_pyprompter=no
-fi
+enable_pyprompter=yes
 AM_CONDITIONAL([ENABLE_PYPROMPTER], [test "x$enable_pyprompter" = "xyes"])
 
 
